### PR TITLE
New Rule: Attachment PDF single link to zip no creator

### PR DESCRIPTION
**Pull request by morriscode**
- New Rule: Attachment: PDF link to zip no creator
- Renaming file
- Deleting old file
- Removing dev/shm logic adding proper creator line....
😮

### CODE REVIEW

File: detection-rules/attachment_pdf_link_to_zip_no_creator.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+description: "Detects a PDF attachment with a single link to a ZIP file. Bumblebee delivery mechanism"
+type: "rule"
+severity: "high"
+source: |
+  type.inbound 
+  and all(attachments, 
+    .file_extension == "pdf"
+    and any(file.explode(.), (
+      .depth == 0 
+      and length(filter(.scan.strings.strings, regex.contains(.,'\/S\/URI\/URI\(http(s):\/\/.+\.zip'))) == 1
```
Comment (jkamdjou) [inline detection-rules/attachment_pdf_link_to_zip_no_creator.yml:11]: i'm guessing the URL scanner isn't pulling this out reliably?

**Comment by jkamdjou** [comment]
> * Renaming file
> * Deleting old file
> * Removing dev/shm logic adding proper creator line....
>   😮

did you mean to commit the deletion of an existing rule?

File: detection-rules/attachment_pdf_link_to_zip_no_creator.yml (hunk 1 above)
Comment (morriscode) [reply detection-rules/attachment_pdf_link_to_zip_no_creator.yml:11]: ScanUrl doesn't run on PDFs unfortunately. This is resolved in the new scanner, but I don't think we're ready to incorporate that yet.

**Comment by morriscode** [comment]
> 

Haha, yes, so I meant to just replace the file I put in there with the corrected one, but the one I put in there was named differently. Cascading failures on my part but this is right now.

**Review by jkamdjou — APPROVED** [review]

**Pull request merged**